src/components/Facility/LocationManagement.tsx

```diff
@@ -42,6 +45,18 @@ export default function LocationManagement({ facilityId }: Props) {
     name: "",
     id: "",
   });
+  const [facilityMiddleware, setFacilityMiddleware] = useState<
+    string | undefined
+  >(undefined);
+
+  useQuery(routes.getPermittedFacility, {
```

Comment:
Update to use `useQuery` from TanStackQuery. This function is no longer present.

Reply:
looks like many changes are made after this PR, should i merge develop branch to issue branch or create new branch?

Reply:
merge develop to this branch